client/src/Components/Footer.js

```diff
@@ -0,0 +1,94 @@
+import { NavLink } from 'react-router-dom';
+import styled from 'styled-components/macro';
+
+import { ReactComponent as MarketsIcon } from '../Images/markets.svg';
+import { ReactComponent as PortfolioIcon } from '../Images/portfolio.svg';
+import { ReactComponent as NewsIcon } from '../Images/news.svg';
+
+export default function Footer({ isStatic }) {
+  return (
+    <footer>
+      <NavWrapper isStatic={isStatic}>
+        <NavIcons>
+          <NavLinkStyled exact to="/">
+            <MarketsButton title="Home" role="img" />
+          </NavLinkStyled>
+          <NavLinkStyled to="/Portfolio">
```

Comment:
In the `App.js` you defined the routes with small letters, best keep it the same. Could cause issues.
```suggestion
          <NavLinkStyled to="/portfolio">
```

Reply:
Thanks for the hint!

Reply:
Thanks for the hint! Fixed